swag_runbook.md

```diff
@@ -4,50 +4,107 @@ Use this template to write up a runbook for your area of concern.
 
 ## Role description
 
-What does this role do? What are you primarily responsible for?
+The SWAG role is responsible for all things SWAG
 
-### General Tasks
+This team will be in charge of getting swags for attendees, speakers, and organizers/volunteers.
+This includes badges, lanyards, t-shirts, and gifts for speakers/organizers/volunteers.
 
-Can you name typical tasks? For example:
+### General Tasks
 
-* Write a compelling CFP (the CFP committee)
-* Requisition a unique annual logo (swag)
-* etc.
+* Coordinating the creation of the logo
+  - **Need to know the theme**
+  - draft artist search.  
+  - search for artist(s)
+  - contract
+  - artist relations
+  - base designs for the event
+* Attendee shirts (typical 1 unique shirt per day)
+  - Select vendor, designs for t-shirts for attendees
+* Organizer shirts
+  - Select vendor, designs for t-shirts for organizers/volunteers
+* Speaker gifts
+  - Select vendor & gift for speakers (no more cups)
+* Attendee badges
+  - Select vendor, designs for badges (note: get nicer ones so its a keepsake, recommended by Karthik)
+* Sponsor Lanyards
+  - Select vendor, designs for lanyard
 
 ## Important Dates
 
 What dates are important for this role? The following headings and details are examples.
 
 ### In the beginning
 
-* Come up with good stage jokes.
+* What is the theme for the conference.  This will influence the logo
 
 ### 3 Months Out
 
-* Find vendors for your stage costume.
+* Start the logo process.
+* Identify candidates to create the logo
+* Find vendors for shirts & gifts
+* Find and work with artists (artist search post and contract)
+* Review budget & requirements (# of attendees)
 
 ### 2 Months Out
 
-* Try on the costume.
+* Meet to finalize the following
+  * Artist(s) and the designs **need this to move forward with the rest**
+  * Speaker gifts (Identify production time)
+  * Start engage vendors on pricing based on expected attendance.
+    * Shirts
+      * Day 1 and Day 2
+      * Organizers
+      * Volunteers
+    * Speaker gifts
+    * Start designing the badge. Usually the logo is involved and a double sided badge with name on both sides is preferable.  Typically the vendor will help with the desogn process.
+    * Start the lanyard process based on expected number of attendees.  The lanyards must at a minimum have the sponsors logo and it is nice to have the DevOpsDaysAustin logo.  A lot of time was spent back and forth about getting the DevOpsDaysAustin logo on the lanyard.
```

Comment:
I like having this here since it instructions to consider right in the beginning.  Additionally, there is references to this already in the Reminders or Common Mistakes section.